core/mm/core_mmu.c

```diff
@@ -642,6 +643,10 @@ uint32_t core_mmu_type_to_attr(enum teecore_memtypes t)
 	case MEM_AREA_NSEC_SHM:
 		return attr | TEE_MATTR_PRW | cached;
 	case MEM_AREA_EXT_DT:
+	/* If CFG_DT_SECURE is disabled, fall back to non-secure mapping. */
+#ifdef CFG_DT_SECURE
```

Comment:
Give how this is used I'd prefer a bit more elaborate name, perhaps `CFG_MAP_EXT_DT_SECURE`.

Reply:
Done